coil-base/src/main/java/coil/decode/EmptyDecoder.kt

```diff
@@ -14,10 +14,7 @@ import okio.blackholeSink
  */
 internal object EmptyDecoder : Decoder {
 
-    private val result = DecodeResult(
-        drawable = EMPTY_DRAWABLE,
-        isSampled = false
-    )
+    private val result = DecodeResult(EMPTY_DRAWABLE, false)
```

Comment:
Could the empty decode result be a static singleton, in case someone needed to check against it?

Reply:
It's actually already a singleton since `EmptyDecoder` is an object. Also `result` is never returned publicly so it should be safe to keep private.